Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -1563,7 +1563,7 @@ inline void BuilderCustom::applyValueContent(BuilderState& builderState, CSSValu
 
     bool didSet = false;
     for (auto& item : downcast<CSSValueList>(value)) {
-        if (is<CSSImageGeneratorValue>(item) || is<CSSImageSetValue>(item) || is<CSSImageValue>(item)) {
+        if (is<CSSImageValue>(item) || is<CSSImageSetValue>(item) || item->isImageGeneratorValue()) {
```

Comment:
Should this use the `isImageShape` function?

Reply:
Going to replace isImageShape and these checks with a single isImage() function (see above).